firefox_puppeteer/tests/test_software_update.py

```diff
@@ -9,6 +11,7 @@
 from firefox_puppeteer.api.software_update import SoftwareUpdate
 
 
+@unittest.skip('Bug 1196006 - about and page_info window tests time out on Travis Linux worker')
```

Comment:
Please remove all changes to this file. Skipping should happen in the other bug. But if this alone doesn't help lets skip all the hanging tests.